docs/api-reference.md

```diff
@@ -102,3 +103,23 @@ A function to unmount the test component. This is commonly used to trigger clean
 
 This is the same [`act` function](https://reactjs.org/docs/test-utils.html#act) that is exported by
 `react-test-renderer`.
+
+---
+
+## `cleanup`
+
+```js
+function cleanup: Promise<void>
+```
+
+Unmounts any rendered hooks rendered with `renderHook`, ensuring all effects have been flushed.
+
+> Please note that this is done automatically if the testing framework you're using supports the
+> `afterEach` global (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups
+> after each test.
```

Comment:
❓ Do you happen to have any reference or example of a testing library auto adding a global `afterEach`?

I think I like this, i'm just curious about community best practices and such.

Reply:
The registration of this is lifted almost verbatim from [`@testing-library/react-testing-library#430`](https://github.com/testing-library/react-testing-library/pull/430)

I don't think this is a commonly used pattern (yet).